debian/control

```diff
@@ -20,6 +20,7 @@ Build-Depends:
  libgsystem-dev,
  libnm-dev (>= 1.2.0),
  libostree-dev (>= 2016.15+dev24.a109440),
+ ostree-tests (>= 2017.6),
```

Comment:
Nitpick - can you put this at the end after `ostree` to maintain the alphabetical order?